templates/website/src/components/Media/ImageMedia/index.tsx

```diff
@@ -50,7 +50,7 @@ export const ImageMedia: React.FC<MediaProps> = (props) => {
     src = `${getClientSideURL()}${url}`
   }
 
-  const loading = loadingFromProps || 'lazy'
+  const loading = loadingFromProps || (priority ? 'eager' : 'lazy')
```

Comment:
There is no need to set loading to `eager` here, as this is the default behavior: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#eager. 

```suggestion
  const loading = loadingFromProps || (!priority ? 'lazy' : undefined)
```